Files/Strings/en-US/Resources.resw

```diff
@@ -585,16 +588,4 @@
   <data name="BaseLayoutItemContextFlyoutSetAsLockscreenBackground.Text" xml:space="preserve">
     <value>Set as lockscreen background</value>
   </data>
-  <data name="StatusBarControlGridViewLarge.Text" xml:space="preserve">
```

Comment:
Can you revert the changes to these strings?

Reply:
Fixed by 301387b76bc15c19b091335177b1225e179fcbc8